### DISCUSSION TITLE
Bot cannot start

### DISCUSSION THREAD

**Discussion by cubixty** [discord-tickets/bot#287] [Support (Q&A)]
![image_2022-04-20_122915339](https://user-images.githubusercontent.com/70766921/164220995-5f78e0ad-c163-4ee1-bcb1-cfb3f4ec9e87.png)

Ye so basically this I cant start the bot because of this error . Any way to fix it?

**Comment by conditionals** [comment]
Are you on the latest version of node? 
Run node -v in terminal.
